.github/workflows/publish-release.yml

```diff
@@ -1,67 +1,17 @@
-                ./gradlew build -x check -x test
-            - name: Create lib directory if not exists
-              run: mkdir -p ballerina/lib
-            - name: Run Trivy vulnerability scanner
-              uses: aquasecurity/trivy-action@master
-              with:
-                scan-type: 'rootfs'
-                scan-ref: '/github/workspace/ballerina/lib'
-                format: 'table'
-                timeout: '10m0s'
-                exit-code: '1'
-            - name: Set version env variable
-              run: echo "VERSION=$((grep -w 'version' | cut -d= -f2) < gradle.properties | rev | cut --complement -d- -f1 | rev)" >> $GITHUB_ENV
-            - name: Pre release dependency version update
-              env:
-                  GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
-              run: |
-                  echo "Version: ${VERSION}"
-                  git checkout -b release-${VERSION}
-                  sed -i 's/ballerinaLangVersion=\(.*\)-SNAPSHOT/ballerinaLangVersion=\1/g' gradle.properties
-                  sed -i 's/ballerinaLangVersion=\(.*\)-[0-9]\{8\}-[0-9]\{6\}-.*$/ballerinaLangVersion=\1/g' gradle.properties
-                  sed -i 's/stdlib\(.*\)=\(.*\)-SNAPSHOT/stdlib\1=\2/g' gradle.properties
-                  sed -i 's/stdlib\(.*\)=\(.*\)-[0-9]\{8\}-[0-9]\{6\}-.*$/stdlib\1=\2/g' gradle.properties
-                  git add gradle.properties
-                  git commit -m "Move dependencies to stable version" || echo "No changes to commit"
-            - name: Publish artifact
-              env:
-                  GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
-                  BALLERINA_CENTRAL_ACCESS_TOKEN: ${{ secrets.BALLERINA_CENTRAL_ACCESS_TOKEN }}
-                  packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
-                  packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
-                  publishUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
-                  publishPAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
-              run: |
-                  ./gradlew clean release -Prelease.useAutomaticVersion=true
-                  ./gradlew -Pversion=${VERSION} publish -x test -PpublishToCentral=true
-            - name: GitHub Release and Release Sync PR
-              env:
-                  GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
-              run: |
-                gh release create v$VERSION --title "module-ballerina-soap-v$VERSION"
-                gh pr create --base master --title "[Automated] Sync master after $VERSION release" --body "Sync master after $VERSION release"
+  call_workflow:
+    name: Run Release Workflow
+    if: ${{ github.repository_owner == 'ballerina-platform' }}
+    uses: ballerina-platform/ballerina-standard-library/.github/workflows/release-package-template.yml@main
+    secrets: inherit
+    with:
+      package-name: soap
+      package-org: ballerina
+      additional-build-flags: "-x :soap-examples:build"
```

Comment:
I think we can remove this. we don't have soap-examples sub-project right ?

Reply:
Yes. Currently we don't have soap-examples